<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReconWindow</class>
 <widget class="QWidget" name="ReconWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reconnaissance</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/icons/recon.png</normaloff>../assets/icons/recon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="configWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="targetGroupBox">
         <property name="title">
          <string>Target</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="targetTypeLabel">
            <property name="text">
             <string>Target Type:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="targetTypeComboBox">
            <item>
             <property name="text">
              <string>Domain</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IP Address</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>IP Range</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CIDR Notation</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>URL</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="targetLabel">
            <property name="text">
             <string>Target:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="targetLineEdit">
            <property name="placeholderText">
             <string>Enter domain, IP, IP range, CIDR, or URL</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="targetFileLabel">
            <property name="text">
             <string>Or Target File:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="targetFileLineEdit">
              <property name="placeholderText">
               <string>Path to file containing targets (one per line)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="browseButton">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="optionsGroupBox">
         <property name="title">
          <string>Reconnaissance Options</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QWidget" name="leftOptionsWidget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="dnsEnumCheckBox">
               <property name="text">
                <string>DNS Enumeration</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="whoisCheckBox">
               <property name="text">
                <string>WHOIS Lookup</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="subdomainEnumCheckBox">
               <property name="text">
                <string>Subdomain Enumeration</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="reverseIPCheckBox">
               <property name="text">
                <string>Reverse IP Lookup</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="rightOptionsWidget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="portScanCheckBox">
               <property name="text">
                <string>Port Scanning</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="portRangeLabel">
                 <property name="text">
                  <string>Port Range:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="portRangeLineEdit">
                 <property name="text">
                  <string>1-1024</string>
                 </property>
                 <property name="placeholderText">
                  <string>e.g., 1-1024,3389,8080</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="serviceDetectionCheckBox">
               <property name="text">
                <string>Service Detection</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="osDetectionCheckBox">
               <property name="text">
                <string>OS Detection</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="advancedOptionsWidget" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QCheckBox" name="bannerGrabbingCheckBox">
               <property name="text">
                <string>Banner Grabbing</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="tracerouteCheckBox">
               <property name="text">
                <string>Traceroute</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="sslInfoCheckBox">
               <property name="text">
                <string>SSL/TLS Information</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="screenshotCheckBox">
               <property name="text">
                <string>Take Screenshots</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="advancedGroupBox">
         <property name="title">
          <string>Advanced Settings</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="threadsLabel">
            <property name="text">
             <string>Threads:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="threadsSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="timeoutLabel">
            <property name="text">
             <string>Timeout (sec):</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="timeoutSpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="wordlistLabel">
            <property name="text">
             <string>Subdomain Wordlist:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QComboBox" name="wordlistComboBox">
              <item>
               <property name="text">
                <string>Default</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Small</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Medium</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Large</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Custom</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="customWordlistLineEdit">
              <property name="placeholderText">
               <string>Path to custom wordlist</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="browseWordlistButton">
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Start Recon</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../assets/icons/start.png</normaloff>../assets/icons/start.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../assets/icons/stop.png</normaloff>../assets/icons/stop.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Save Results</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../assets/icons/save.png</normaloff>../assets/icons/save.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="resultsTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="summaryTab">
       <attribute name="title">
        <string>Summary</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTextEdit" name="summaryTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dnsTab">
       <attribute name="title">
        <string>DNS Records</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QTableView" name="dnsTableView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="subdomainsTab">
       <attribute name="title">
        <string>Subdomains</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QTableView" name="subdomainsTableView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="portsTab">
       <attribute name="title">
        <string>Ports</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QTableView" name="portsTableView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="whoisTab">
       <attribute name="title">
        <string>WHOIS</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QTextEdit" name="whoisTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="servicesTab">
       <attribute name="title">
        <string>Services</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QTableView" name="servicesTableView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="rawTab">
       <attribute name="title">
        <string>Raw Output</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QTextEdit" name="rawTextEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Ready</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>